R/correlationbayesian.R

```diff
@@ -660,8 +661,19 @@ CorrelationBayesianInternal <- function(jaspResults, dataset=NULL, options, ...)
 
   labels <- if (useRanks) .addRankToVariableName(vars) else vars
 
-  obj <- try(jaspGraphs::ggMatrixPlot(plotList = plotMat, leftLabels = labels, topLabels = labels,
-                                      scaleXYlabels = 0.9, labelPos=labelPos))
+  if (nVariables > 2 || options[["matrixPlotDensity"]] || options[["matrixPlotPosterior"]]) {
```

Comment:
Why not also bring this to the frequentist analysis?

Reply:
OK just added it